Components/modal.css

```diff
@@ -0,0 +1,120 @@
+.modal-container {
+    width: 100%;
+    height: 100vh;
+    background-color: rgb(187, 187, 245);
```

Comment:
Use var for colour.

Reply:
+1. Also use hex everywhere for consistency.